src/main/java/Numbers.java

```diff
@@ -0,0 +1,60 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Numbers {
+
+    private final List<Number> numbers;
+
+    public Numbers(Input input) {
+        this.numbers = createNumbers(input);
+    }
+
+    public Numbers(Number... numbers) {
+        /* For Test */
+        this.numbers = new ArrayList<>();
+        this.numbers.addAll(Arrays.asList(numbers));
+    }
+
+
+    public Number calculateAddAll() {
+        Number result = new Number(0);
+
+        for(Number number : numbers) {
+            result = result.plus(number);
+        }
+
+        return result;
+    }
+
+    private List<Number> createNumbers(Input input) {
```

Comment:
```suggestion
    private List<Number> createNumbers(String[] inputs) {
```

Input 객체를 넘기는 것 보다 요청온 데이터를 넘겨주는 것이 어떨까요?

Reply:
혹시 이유를 알수 있을까요 ~? 제가 잘 몰라서요~!

Reply:
`Input` 객체에 의존적일 수가 있습니다~
다른 객체의 의존을 제거해서 일급컬렉션인 Numbers 의 응집도를 높이고 결합도를 낮추는 것이 좋은 객체지향의 설계 방법 중 하나 입니다~

Reply:
아하 음. 네 ! 감사합니다~ 제가 일급컬렉션 부분을 더 공부해보겠습니다~!

Reply:
지금과 같이 제가 리뷰를 진행 중일 땐 메일을 바로 확인할 수 있으나!! 제가 업무 중일 땐 가끔 놓칠 때가 있어요~
그럴 땐 코멘트 남겨주시고 슬랙으로 DM 주시면 바로 확인이 가능합니다 :)

Reply:
네네 감사합니다 그러겠습니다~! ^-^